### ISSUE TITLE
Offline search is not working anymore

### ISSUE THREAD

**Issue by alexrashed** [google/docsy#523]
Hi there!

With the latest commit (4c82fce351745e21bc59bb9db09bfc86e88de3b2), the offline search is not working anymore for me.
The console logs the following exceptions:
```
jQuery.Deferred exception: $(...).tooltip is not a function @http://localhost:1313/js/main.js:22:38
e@https://code.jquery.com/jquery-3.5.1.min.js:2:30005
l/</t<@https://code.jquery.com/jquery-3.5.1.min.js:2:30307
setTimeout handler*l/<@https://code.jquery.com/jquery-3.5.1.min.js:2:30516
c@https://code.jquery.com/jquery-3.5.1.min.js:2:28294
fireWith@https://code.jquery.com/jquery-3.5.1.min.js:2:29039
fire@https://code.jquery.com/jquery-3.5.1.min.js:2:29075
c@https://code.jquery.com/jquery-3.5.1.min.js:2:28294
fireWith@https://code.jquery.com/jquery-3.5.1.min.js:2:29039
ready@https://code.jquery.com/jquery-3.5.1.min.js:2:32012
B@https://code.jquery.com/jquery-3.5.1.min.js:2:31791
EventListener.handleEvent*@https://code.jquery.com/jquery-3.5.1.min.js:2:32160
@https://code.jquery.com/jquery-3.5.1.min.js:2:220
@https://code.jquery.com/jquery-3.5.1.min.js:2:225
 undefined jquery-3.5.1.min.js:2:31560

jQuery.Deferred exception: $targetSearchInput.popover is not a function render@http://localhost:1313/js/main.js:193:32
@http://localhost:1313/js/main.js:143:19
dispatch@https://code.jquery.com/jquery-3.5.1.min.js:2:43090
add/v.handle@https://code.jquery.com/jquery-3.5.1.min.js:2:41074
trigger@https://code.jquery.com/jquery-3.5.1.min.js:2:71513
trigger/<@https://code.jquery.com/jquery-3.5.1.min.js:2:72108
each@https://code.jquery.com/jquery-3.5.1.min.js:2:2976
each@https://code.jquery.com/jquery-3.5.1.min.js:2:1454
trigger@https://code.jquery.com/jquery-3.5.1.min.js:2:72084
@http://localhost:1313/js/main.js:187:30
e@https://code.jquery.com/jquery-3.5.1.min.js:2:30005
l/</t<@https://code.jquery.com/jquery-3.5.1.min.js:2:30307
setTimeout handler*l/<@https://code.jquery.com/jquery-3.5.1.min.js:2:30516
c@https://code.jquery.com/jquery-3.5.1.min.js:2:28294
fireWith@https://code.jquery.com/jquery-3.5.1.min.js:2:29039
fire@https://code.jquery.com/jquery-3.5.1.min.js:2:29075
c@https://code.jquery.com/jquery-3.5.1.min.js:2:28294
fireWith@https://code.jquery.com/jquery-3.5.1.min.js:2:29039
l@https://code.jquery.com/jquery-3.5.1.min.js:2:79800
o/<@https://code.jquery.com/jquery-3.5.1.min.js:2:82254
EventHandlerNonNull*send@https://code.jquery.com/jquery-3.5.1.min.js:2:82440
ajax@https://code.jquery.com/jquery-3.5.1.min.js:2:78223
@http://localhost:1313/js/main.js:162:11
e@https://code.jquery.com/jquery-3.5.1.min.js:2:30005
l/</t<@https://code.jquery.com/jquery-3.5.1.min.js:2:30307
setTimeout handler*l/<@https://code.jquery.com/jquery-3.5.1.min.js:2:30516
c@https://code.jquery.com/jquery-3.5.1.min.js:2:28294
fireWith@https://code.jquery.com/jquery-3.5.1.min.js:2:29039
fire@https://code.jquery.com/jquery-3.5.1.min.js:2:29075
c@https://code.jquery.com/jquery-3.5.1.min.js:2:28294
fireWith@https://code.jquery.com/jquery-3.5.1.min.js:2:29039
ready@https://code.jquery.com/jquery-3.5.1.min.js:2:32012
B@https://code.jquery.com/jquery-3.5.1.min.js:2:31791
EventListener.handleEvent*@https://code.jquery.com/jquery-3.5.1.min.js:2:32160
@https://code.jquery.com/jquery-3.5.1.min.js:2:220
@https://code.jquery.com/jquery-3.5.1.min.js:2:225
 undefined jquery-3.5.1.min.js:2:31560
```

When using the commit before (05ef33bf2067c979b273bcee104a45b1e54e21b4), the offline search is working and the exceptions are not thrown / logged.

The issue can be reproduced with the following repo: [alexrashed/docsy-offline-search](https://github.com/alexrashed/docsy-offline-search)
The repo has been created with the following steps:
- Create a new fork with the official docsy template in GitHub.
- Disable google search and enable offline search.
- Update the submodule reference to the latest commit.

Let me know if I can be of service when analyzing the issue.

PS.: Thanks for all the awesome work!

**Comment by narrenfrei** [comment]
Maybe it's also a side effect of the bootstrap update. If so, there exists already a (not merged) bug fix: #522

You could try to change the SRI hash as done in #522

Can't give more help at the moment, because at the moment I'm only on my mobile.

**Comment by LisaFC** [comment]
Fixed by #522
